samples/tab-add-in-combined/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,167 @@
+{
+    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+    "manifestVersion": "devPreview",
```

Comment:
Is this feature is in devPreview or we should use public version of manifest?

Reply:
Will cover in different new PR